RELEASES.md

```diff
@@ -1,5 +1,17 @@
 # Release Notes
 
+## [v0.0.36] - 2019-12-02
+
+#### Bug Fixes
+
+*   updated for Bazel 1.2.1 (#4230) ([d039eb79](https://github.com/kythe/kythe/commit/d039eb798ea41128755387cfefae94ee7cd62a62))
```

Comment:
We probably don't need to include this in the release notes.

Reply:
Removed.